Content.Shared/CCVar/CCVars.cs

```diff
@@ -1045,6 +1045,12 @@ public sealed class CCVars : CVars
         public static readonly CVarDef<bool> FlavorText =
             CVarDef.Create("ic.flavor_text", false, CVar.SERVER | CVar.REPLICATED);
 
+        /// <summary>
+        /// Adds a period at the end of a sentence if the sentence ends in a letter.
+        /// </summary>
+        public static readonly CVarDef<bool> ChatPunctuation =
+            CVarDef.Create("ic.punctuation", false, CVar.SERVER | CVar.REPLICATED);
```

Comment:
```suggestion
            CVarDef.Create("ic.punctuation", false, CVar.SERVER);
```

I'm pretty sure this doesn't have to be replicated, as the CVar is only written to or read server side.